src/Pages/Components/GamePage/WinField.tsx

```diff
@@ -0,0 +1,29 @@
+import { observer } from 'mobx-react';
+import { gameStore } from '../../../Stores/GameStore';
+
+import RestartButton from '../../../StyledComponents/RestartButton';
+
+const WinField = observer(() => {
+
+	const isGameEnded = gameStore.GameState.IsGameEnded == undefined;
+
+	if (gameStore.GameState.IsGameEnded) {
+		return (
+			<>
+				<p>{gameStore.GameState.WinnerName} Win!</p>
```

Comment:
Looks like you can sth like
```suggestion
<p>gameStore.GameState.IsGameEnded ? ({gameStore.GameState.WinnerName} Win!) : (Draw!)</p>
```